docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5472,6 +5472,78 @@ the purpose of adding features for the P4Runtime API.
 | `@p4runtime_translation` | See sections [#sec-user-defined-types], [#sec-translation-of-port-numbers] |
 +--------------------------+---------------------------------------+
 ~
+### Custom Annotations in P4Info
+P4 allows flexible annotations. Users may add their own annotations as long as they obey the language syntax.
```

Comment:
I suggest adding a reference to the P4_16 spec here: https://p4.org/p4-spec/docs/P4-16-v1.1.0-spec.html#sec-annotations

Reply:
Item 1 - repeated expressions - great idea, I'll make the change. Also, I'm removing the  `empty` field because the container `AnnotationBody` can simply be empty. I'm removing the `oneof` to allow for both expressions and kv-pairs in an annotation body, even though P4-16 currently precludes it. The `expressions` can simply be a `repeated string`, no need for a container now that `oneof` is gone. Also removing the `KVmap` message and renaming to `properties` since that is a more conventional term.
```
// A parsed representation of the annotation body.
// Ex. empty annotation:           @my_anno1()
// Ex. ExpressionList annotation:  @my_anno2(1+1, "Hello", 1 << 3)
// Ex. KV-pair annotation:         @my_anno3(a=b,c=d)
// Ex. Cumulative kv-pair          @my_anno3(e=f)  // adds to my_anno3's properties
message AnnotationBody {
  repeated string expressions = 1;
  map <string,string> properties = 2;
}
```

Item 2 - reference - will do, great idea.